app_flutter/lib/widgets/task_icon.dart

```diff
@@ -44,22 +44,19 @@ class TaskIcon extends StatelessWidget {
           color: blendFilter,
         );
       case StageName.luci:
-        return Image.asset(
-          'assets/chromium.png',
-        );
-      case StageName.devicelab:
-        return Image.asset(
-          'assets/android.png',
```

Comment:
Is there a builder pattern to show android tests?

Reply:
There are android on windows, linux, etc. So not sure if having an android icon for all those platforms would be helpful. Ideally we would want a distinctive win+android, win+linux, etc. icon for those cases.

Reply:
There's better signal when differentiating by device than host (we can reuse the android logo for mac-android, windows-android, and linux-android)

Reply:
done